ui/storybook/src/config/preview.ts

```diff
@@ -101,4 +102,6 @@ setThemeSwitcher(async (theme, channel) => {
   });
 });
 
-export const decorators = [WithTimeZone, WithBackground, WithThemes];
+initializeMSW({ onUnhandledRequest: 'bypass' });
```

Comment:
This `bypass` option disables a default behavior where msw does an annoying console.warn for every query it sees that isn't mocked.